sourcegraph/sourcegraph.proto

```diff
@@ -665,11 +668,22 @@ message TagList {
 	ListResponse list_response = 2 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
 }
 
+message MirrorReposRefreshVCSOp {
+	RepoSpec repo = 1 [(gogoproto.nullable) = false];
+	VCSCredentials credentials = 2;
+}
+
+// VCSCredentials for authentication during communication with VCS remotes.
+message VCSCredentials {
+	// Pass is the password provided to the VCS.
+	string pass = 1 [(gogoproto.nullable) = false];
```

Comment:
I initially started with having both `user` and `pass` fields here. However, `user` ended up being unused. So I removed it because it was unused.

If it's ever used/needed in the future, it can/should be added then.